src/index.css

```diff
@@ -23,16 +28,19 @@
     font-size: xx-large;
     color: aliceblue;
   }
-  .container button {
-    position: absolute;
-    bottom: 10%;
-    right: 1%;
-    transform: translate(-50%, -50%);
-  }
+
   .container h3 {
     position: absolute;
     bottom: 5%;
     left: 10%;
     font-size: x-large;
     color: aliceblue;
-  }
\ No newline at end of file
+  }
+  .container .input-group {
+    width: 400px;
+    bottom: 350px;
+    margin-top: 20px;
+    
+  }
+
```

Comment:
Avoid using global styles to style specific components, as this may lead to conflicts and limit your flexibility. It's better to create small styles.css file for each component and only add styles specific to it. Then the global styles will only be for things you want to apply to all pages of your app.

Reply:
Ok sir